lib/const.rb

```diff
@@ -72,15 +72,22 @@
   CREW_BUILD = 'x86_64-cros-linux-gnu'
 end
 
+CREW_COMMON_FLAGS = "'-Os -pipe -flto=auto -fuse-ld=gold'"
+CFLAGS = CREW_COMMON_FLAGS
+CXXFLAGS = CREW_COMMON_FLAGS
+FCFLAGS = CREW_COMMON_FLAGS
+FFLAGS = CREW_COMMON_FLAGS
```

Comment:
No need to create all these constants.  Just use `CREW_COMMON_FLAGS` for everything.

Reply:
What if we need to individualize these?

Reply:
We cross that bridge when we get there.  Conflict in `lib/const.rb`?

Reply:
I went ahead and resolved the conflict.

Reply:
Fixed.